.github/workflows/redis-server-tests.yml

```diff
@@ -24,8 +24,6 @@ jobs:
           - 3
           - 4
           - 5
-          - 6
```

Comment:
@tillkruss We don't need this versions in this workflow, we're running tests through `latest` version of server within our `redis-stack` workflow, so we make sure that we have all modules available and we can run tests for module commands